classes/02class/exercises/c02-network03/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,71 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```
+$ aws ec2 create-vpc \
+    --cidr-block 10.11.0.0/16
```

Comment:
would this command include a ipv6 cidr block as well?

Reply:
It doesn't associate IPv6 CIDR blocks with the VPC. The command below achieves the same result, but a bit more 'explicit':

`$ aws ec2 create-vpc \
    --cidr-block 10.11.0.0/16 \
    **--no-amazon-provided-ipv6-cidr-block**`